_includes/tutorials/kafka-producer-application-callback/confluent/markup/dev/make-dev-file.adoc

```diff
@@ -0,0 +1,21 @@
+Then create a development file at `configuration/dev.properties`:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-producer-application-callback/confluent/code/configuration/dev.properties %}</code></pre>
++++++
+
+Let's do a quick walkthrough of some of the producer properties.
```

Comment:
@awalther28 If it's the case that everything below is the same as the non-CCloud tutorial, can you dedupe (either with an include or a separate adoc file?

Reply:
Yea totally, I split it up into a new doc and added an include for that new doc. I verified the rendering locally. Work was done in commit https://github.com/confluentinc/kafka-tutorials/pull/853/commits/5c580ee2d045b60b81a8f05a59eb6a1ed235ab9f